### PR TITLE
Readonly forms

### PR DESCRIPTION
**Pull request by timbl**
Where the data being viewed by a form is not editable -- either server capability or access control -- this PR disables user input, making a read-only view which is is cleaner, and does not invite editing.

### CODE REVIEW

**Comment by timbl** [comment]
rebased

File: src/signin.js [hunk 1]
```diff
@@ -998,6 +998,7 @@ function loginStatusBox (dom, listener, options) { // 20190630
   }
 
   var logoutButton = function (me, options) {
+    options = options || {}
```
Comment (megoth) [inline src/signin.js]: Not important, but you could've done this in the previous line as well, with `function (me, options = {})`

**Comment by megoth** [comment]
I'll merge this to master after we've done a publish of the latest rdflib, as this is dependent on the latest changes there.

**Comment by megoth** [comment]
(Will probably release new version of rdflib on Sunday)

**Pull request merged**